pkg/asset/agent/agentconfig/agenthosts.go

```diff
@@ -258,6 +252,11 @@ func (a *AgentHosts) getInstallConfigDefaults(hosts []*baremetal.Host) error {
 			host.Interfaces = append(host.Interfaces, hostInterface)
 		}
 
+		// Add BMC configuration
+		host.BMC = icHost.BMC
+
+		installConfig.ValidateBMCConfiguration()
```

Comment:
I wasn't expecting to find a validation check here, but rather in `validateAgentHosts`

Reply:
Moved.